gatorgrade/input/test_in_file_path.py

```diff
@@ -0,0 +1,17 @@
+'''
```

Comment:
This test suite should be in `tests/input/`, instead of `gatorgrade/input`.

Reply:
Alright. I'll delete the test.py file